cpp/open3d/t/io/file_format/FilePLY.cpp

```diff
@@ -140,10 +140,34 @@ static std::tuple<std::string, int, int> GetNameStrideOffsetForAttribute(
     if (name == "green") return std::make_tuple("colors", 3, 1);
     if (name == "blue") return std::make_tuple("colors", 3, 2);
 
+    // Extra attributes for 3DGS.
+    if (name.rfind("f_dc_", 0) == 0) {
+        int offset = std::stoi(name.substr(5));
```

Comment:
Maybe add some documentation here of what these properties and the reason of those numbers are